packages/codeeditor/src/editor.ts

```diff
@@ -451,10 +451,18 @@ namespace CodeEditor {
 
     /**
      * Repaint the editor.
+     * 
+     * #### Notes
+     * A repainted editor should fit to its host node.
      */
     refresh(): void;
 
     /**
+     * Resize the editor to fit its host node.
+     */
+    resizeToFit(): void;
```

Comment:
Could we? `CodeMirrorEditor` is not `Widget`, it is not notified about show/attach events.